joomla/joomla-cms
# [4.2] Indicate if category not published

Articles that are in a category which is unpublished or trashed are not visible on the site but there is no way you can tell this from the article manager

### Summary of Changes
1. Adds an indicator if the category is unpublished or trashed. 
2. Changes the published icon to a disabled icon and adds a tooltip to the icon

### Testing Instructions
Create a catgory called test and create 4 featured articles in that category. 
Unpublish or Trash the category

Check the list of articles and the list of featured articles

### Actual result BEFORE applying this Pull Request
No indication that the article can not be displayed


### Expected result AFTER applying this Pull Request
Text as shown in the image below

![image](https://user-images.githubusercontent.com/1296369/164726672-f9bf7267-18e4-4b88-b762-3bd93d2cb83a.png)


### Documentation Changes Required
